docs/Building_Diagrams/pools_and_lanes.md

```diff
@@ -70,3 +70,52 @@ Remember that each pool requires Lane configuration, even if it contains just a
 | --- | --- | --- |
 | ![participant_sales](images/participant_lane_1.png) | **Name:** Manager | A concise and descriptive label that accurately represents the owner and role of the Lane. |
 | ![data_object_pools](images/data_object_pools_1.png) | **ID:** lane_manager | A distinct ID to differentiate each Lane, especially when there are multiple. |
+
+### Example: Using Lanes and Pools for Petty Cash Request Process
+This example demonstrates the application of Lanes and pools in a BPMN diagram, specifically designed to handle a petty cash request process within an organization. 
+
+The process is structured around different tasks allocated to lane and pools, emphasizing role-based access and task execution.
+
+
+#### BPMN Diagram
+![Lanes and Pools Example](images/lanes_pools_example_1.png)
+
+**Process Flow:**
+
+1. **Start Event**: The workflow kicks off with a start event signaling the initiation of a petty cash request.
+
+2. **User Task: Petty Cash Request**: This task uses a form to collect petty cash requests, including the requested amount and the reason for the request.
+
+    ![Lanes and Pools Example](images/lanes_pools_example_2.png)
+
+3. **Script Task: Assign Lanes** : This script dynamically designates the lane owner for the next task based on the user's role, ensuring the "Cashier" is set as the lane owner for subsequent approval tasks.
+
+
+    **Script**:
+     ```python
```

Comment:
@usama9500 can you please avoid using lane_owners and just assume that the task gets assigned to a group based on the lane name? if you need to document lane_owners specifically, avoid get_current_user and just put someone's name in there, like "jim" rather than using code to do it. but in this case, i think just remove 3. **Script Task: Assign Lanes**  and lane_owners.

the reason i say this is because you are not allowed to call get_current_user outside a human task.

Reply:
> @usama9500 can you please avoid using lane_owners and just assume that the task gets assigned to a group based on the lane name? if you need to document lane_owners specifically, avoid get_current_user and just put someone's name in there, like "jim" rather than using code to do it. but in this case, i think just remove 3. **Script Task: Assign Lanes** and lane_owners.
> 
> the reason i say this is because you are not allowed to call get_current_user outside a human task.

Got it Updating it. Thanks.